lang/main.json

```diff
@@ -250,7 +250,7 @@
     "dialog": {
         "Back": "Back",
         "Cancel": "Cancel",
-        "IamHost": "Log-in",
+        "IamHost": "Sign in",
```

Comment:
@damencho @saghul I suggest we switch to sign in and sign out for action buttons because this is more consistent regarding the fact that we sign in via google, github or facebook.

Reply:
The terminology was discussed with Emil, please don't change it.

Reply:
Ok. What should the action buttons on ProfileView read? Log-in Log-out?

Reply:
Yes please!

Reply:
Perfect! Updated!